--- config.def.h	2020-04-05 06:09:33.227683067 -0400
+++ config.h	2020-04-05 06:10:22.055087407 -0400
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "monospace:pixelsize=12:antialias=true:autohint=true";
+static int borderpx = 0;
 static const char *resourceFiles[] = { /* Better Xresources */
 	"~/.config/xconf/Xcolor",
 	"~/.config/xconf/Xresources",
@@ -222,25 +222,34 @@ static MouseShortcut mshortcuts[] = {
 /* Internal keyboard shortcuts. */
 #define MODKEY Mod1Mask
 #define TERMMOD (ControlMask|ShiftMask)
+#define CAPS LockMask
+#define CTRL ControlMask
+#define SHFT ShiftMask
+#define ALT  Mod1Mask
+#define NLCK Mod2Mask
+#define MENU Mod3Mask
+#define SUPR Mod5Mask
 
 static Shortcut shortcuts[] = {
 	/* mask                 keysym          function        argument */
 	{ XK_ANY_MOD,           XK_Break,       sendbreak,      {.i =  0} },
-	{ ControlMask,          XK_Print,       toggleprinter,  {.i =  0} },
-	{ ShiftMask,            XK_Print,       printscreen,    {.i =  0} },
+	{ CTRL,                 XK_Print,       toggleprinter,  {.i =  0} },
+	{ SHFT,                 XK_Print,       printscreen,    {.i =  0} },
 	{ XK_ANY_MOD,           XK_Print,       printsel,       {.i =  0} },
-	{ TERMMOD,              XK_Prior,       zoom,           {.f = +1} },
-	{ TERMMOD,              XK_Next,        zoom,           {.f = -1} },
-	{ TERMMOD,              XK_Home,        zoomreset,      {.f =  0} },
-	{ TERMMOD,              XK_C,           clipcopy,       {.i =  0} },
-	{ TERMMOD,              XK_V,           clippaste,      {.i =  0} },
-	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
-	{ ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
-	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
-	{ TERMMOD,              XK_R,           loadxresources, {.i =  0} }, /* Better Xresources */
-	{ TERMMOD,              XK_I,           iso14755,       {.i =  0} }, /* ISO14755 */
-	{ ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} }, /* Scrollback */
-	{ ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} }, /* Scrollback */
+	{ CTRL|SHFT,            XK_Page_Up,     zoom,           {.f = +1} },
+	{ CTRL|SHFT,            XK_Page_Down,   zoom,           {.f = -1} },
+	{ CTRL|SHFT,            XK_Home,        zoomreset,      {.f =  0} },
+	{ CTRL|SHFT,            XK_C,           clipcopy,       {.i =  0} },
+	{ CTRL|SHFT,            XK_V,           clippaste,      {.i =  0} },
+	{ CTRL|SHFT,            XK_Y,           selpaste,       {.i =  0} },
+	{ SHFT,                 XK_Insert,      selpaste,       {.i =  0} },
+	{ CTRL|SHFT,            XK_Num_Lock,    numlock,        {.i =  0} },
+	{ CTRL|SHFT,            XK_R,           loadxresources, {.i =  0} }, /* Better Xresources */
+	{ CTRL|SHFT,            XK_I,           iso14755,       {.i =  0} }, /* ISO14755 */
+	{ ALT,                  XK_Page_Up,     kscrollup,      {.i = -1} }, /* Scrollback */
+	{ ALT,                  XK_Page_Down,   kscrolldown,    {.i = -1} }, /* Scrollback */
+	{ ALT,                  XK_Up,          kscrollup,      {.i =  1} },
+	{ ALT,                  XK_Down,        kscrolldown,    {.i =  1} },
 };
 
 /*
