diff --git a/config.def.h b/config.h
index 82c8caa..a5fd758 100644
--- a/config.def.h
+++ b/config.h
@@ -5,14 +5,15 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "monospace:pixelsize=12:antialias=true:autohint=true";
 /* Spare fonts */
+/* TODO: Custom fonts don't seem to change? */
 static char *font2[] = {
-/*	"Inconsolata for Powerline:pixelsize=12:antialias=true:autohint=true", */
-/*	"Hack Nerd Font Mono:pixelsize=11:antialias=true:autohint=true", */
+	"monospace2:pixelsize=12:antialias=true:autohint=true",
+	"japanese:pixelsize=12:antialias=true:autohint=true",
+	"math:pixelsize=12:antialias=true:autohint=true",
 };
-
-static int borderpx = 2;
+static int borderpx = 0;
 static const char *resourceFiles[] = { /* Better Xresources */
 	"~/.config/xconf/Xresources",
 };
@@ -224,6 +225,12 @@ ResourcePref resources[] = {
 		{ "rows",           INTEGER, &rows           },
 		{ "cwscale",        FLOAT,   &cwscale        },
 		{ "chscale",        FLOAT,   &chscale        },
+
+		{ "baseAlpha",      FLOAT,   &baseAlpha      },
+		{ "alpha",          FLOAT,   &alpha          },
+		{ "alphaNoFocus",   FLOAT,   &alphaNoFocus   },
+		{ "alpha2",         FLOAT,   &alpha2         },
+		{ "alpha2NoFocus",  FLOAT,   &alpha2NoFocus  },
 };
 /* --Better Xresources */
 
@@ -250,39 +257,48 @@ static MouseShortcut mshortcuts[] = {
 /* Internal keyboard shortcuts. */
 #define MODKEY Mod1Mask
 #define TERMMOD (ControlMask|ShiftMask)
+#define CAPS LockMask
+#define CTRL ControlMask
+#define SHFT ShiftMask
+#define MOD1 Mod1Mask /* Alt */
+#define MOD2 Mod2Mask /* Numlock */
+#define MOD3 Mod3Mask /* Menu */
+#define MOD4 Mod4Mask /* Super */
 
 static Shortcut shortcuts[] = {
 	/* mask                 keysym          function        argument */
 	{ XK_ANY_MOD,           XK_Break,       sendbreak,      {.i =  0} },
-	{ ControlMask,          XK_Print,       toggleprinter,  {.i =  0} },
-	{ ShiftMask,            XK_Print,       printscreen,    {.i =  0} },
+	{ CTRL,                 XK_Print,       toggleprinter,  {.i =  0} },
+	{ SHFT,                 XK_Print,       printscreen,    {.i =  0} },
 	{ XK_ANY_MOD,           XK_Print,       printsel,       {.i =  0} },
-	{ TERMMOD,              XK_Prior,       zoom,           {.f = +1} },
-	{ TERMMOD,              XK_Next,        zoom,           {.f = -1} },
-	{ TERMMOD,              XK_Home,        zoomreset,      {.f =  0} },
-	{ TERMMOD,              XK_C,           clipcopy,       {.i =  0} },
-	{ TERMMOD,              XK_V,           clippaste,      {.i =  0} },
-	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
-	{ ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
-	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
+	{ CTRL|SHFT,            XK_Page_Up,     zoom,           {.f = +1} },
+	{ CTRL|SHFT,            XK_Page_Down,   zoom,           {.f = -1} },
+	{ CTRL|SHFT,            XK_Home,        zoomreset,      {.f =  0} },
+	{ CTRL|SHFT,            XK_C,           clipcopy,       {.i =  0} },
+	{ CTRL|SHFT,            XK_V,           clippaste,      {.i =  0} },
+	{ CTRL|SHFT,            XK_Y,           selpaste,       {.i =  0} },
+	{ SHFT,                 XK_Insert,      selpaste,       {.i =  0} },
+	{ CTRL|SHFT,            XK_Num_Lock,    numlock,        {.i =  0} },
 	/* Better Alpha */
-	{ TERMMOD,              XK_P,           toggleAlphaMode,{.i =  0} },
+	{ CTRL|SHFT,            XK_P,           toggleAlphaMode,{.i =  0} },
 	/* Better Alpha */
-	{ TERMMOD,              XK_A,           toggleAlpha,    {.i =  0} },
+	{ CTRL|SHFT,            XK_A,           toggleAlpha,    {.i =  0} },
 	/* Better Alpha */
-	{ TERMMOD,              XK_Up,          modAlpha,       {.f = +0.05} },
+	{ CTRL|SHFT,            XK_Up,          modAlpha,       {.f = +0.05} },
 	/* Better Alpha */
-	{ TERMMOD,              XK_Down,        modAlpha,       {.f = -0.05} },
+	{ CTRL|SHFT,            XK_Down,        modAlpha,       {.f = -0.05} },
 	/* Better Alpha */
-	{ TERMMOD,              XK_O,           resetAlpha,     {.i =  0} },
+	{ CTRL|SHFT,            XK_O,           resetAlpha,     {.i =  0} },
 	/* Better Xresources */
-	{ TERMMOD,              XK_R,           rloadResources, {.i =  0} },
+	{ CTRL|SHFT,            XK_R,           rloadResources, {.i =  0} },
 	/* ISO14755 */
-	{ TERMMOD,              XK_I,           iso14755,       {.i =  0} },
+	{ CTRL|SHFT,            XK_I,           iso14755,       {.i =  0} },
 	/* Scrollback */
-	{ ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },
+	{ MOD1,                 XK_Page_Up,     kscrollup,      {.i = -1} },
 	/* Scrollback */
-	{ ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} },
+	{ MOD1,                 XK_Page_Down,   kscrolldown,    {.i = -1} },
+	{ MOD1,                 XK_Up,          kscrollup,      {.i =  1} },
+	{ MOD1,                 XK_Down,        kscrolldown,    {.i =  1} },
 };
 
 /*
